analyzer/src/main/resources/init.gradle

```diff
@@ -84,12 +84,12 @@ interface Dependency {
 class DependencyImpl implements Dependency, Serializable {
     String groupId
     String artifactId
-    String version
-    String classifier
-    String extension
+    String version = ''
```

Comment:
Hmm. Comparing this to the `Dependency` class in `GradleSupport.kt`, there only `error` and `localPath` are nullable. So shouldn't we assign default values here to everything else, even if in the current code paths we happen to always assign the other variables? Simply to have a better match between the data structures in Groovy and in Kotlin?